### ISSUE TITLE
sintel training script

### ISSUE THREAD

**Issue by Ecalpal** [visinf/irr#8]
Hi, Junhwa!
I found that the total_epochs of second round in irr/scripts/IRR-PWC_sintel_train_full.sh is 591, Is this correct?
https://github.com/visinf/irr/blob/9a3f66d1ef74ff74679ce876cedd861c04db4eee/scripts/IRR-PWC_sintel_train_full.sh#L63
Does this mean that the second round of training was only 143 epoch and it's learning rate is always 1e-5?

**Comment by hurjunhwa** [comment]
Hi

Yes, that's the stopping point that we found by training on the validation set.
Our model converges faster than the original PWC-Net, so we didn't need to train further.

Yes, it trains 143 epochs with 1e-5.
There can be a better training schedule, but we followed that of PWC-Net to compare the models under the same training setting.

**Comment by Ecalpal** [comment]
Thanks for your detailed answer！

**Comment by Ecalpal** [comment]
Hi, Junhwa!
I still wonder why you stopped at 143 epochs in the second round.
Did you completed two rounds of training and chose the best one which has the lowest epe on the verification set ?

**Comment by hurjunhwa** [comment]
Hi,

(I am referring in terms of # of epochs rather than # of iteration steps)

Here is exactly what we did:
1) FlyingchairsOcc training split: from the scratch to 108th epoch
2) FlyingThings3D training split: from 108th to 159th

Then I tried to find the stopping point with Sintel Train/Valid split:
3) Sintel "Clean + Final" training split (1816 images): from 160th to 489th
4) Sintel Final training split (908 images): from 490th to 1150th

I chose the stopping point in the "4)" step by checking the validation error.

And then I used the all training image for the stopping point found above.
(of course the # of images used for training is different, so the corresponding epochs should be calculated)

5) Sintel  "Clean + Final" all images (2082 images): from 160th to 447th
6) Sintel Final all images (1041 images): from  448th to 591th

Hopefully it makes clear.. :)

**Comment by Ecalpal** [comment]
I got it!
Thank you for your timely answer!
